src/components/stepper/Steps.tsx

```diff
@@ -0,0 +1,196 @@
+import React,{useContext } from "react";
+import { styled } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import { Typography, useTheme } from "@mui/material";
```

Comment:
`Typography` should be imported from `@mui/icons-material/Typography;`

Reply:
import optimized